server/app/services/ambassadors.js

```diff
@@ -154,9 +157,94 @@ async function signup(json, verification, carrierLookup) {
   if (existing_tripler) {
     new_ambassador.relateTo(existing_tripler, "was_once")
   }
+
+  setAmbassadorHubspotID(new_ambassador)
+
   return new_ambassador
 }
 
+/*
+ * setAmbassadorHubspotID(ambassador)
```

Comment:
This does more than set the ID so it would be better with a clearer name like say `connectAmbassadorToHubSpot` or `initialSyncAmbassadorToHubSpot`.

A descriptive comment would also help a lot, e.g.

```
// If the given Ambassador node doesn't already have an `hs_id`:
// (a) populates `hs_id` by asking HubSpot for the HubSpot ID associated
//     with the Ambassador's e-mail address, then
// (b) sends the other data fields on the node over to HubSpot.
```

Reply:
added comment